crates/wallet/src/wallet/coin_selection.rs

```diff
@@ -219,7 +219,7 @@ pub trait CoinSelectionAlgorithm: core::fmt::Debug {
         required_utxos: Vec<WeightedUtxo>,
         optional_utxos: Vec<WeightedUtxo>,
         fee_rate: FeeRate,
-        target_amount: u64,
+        target_amount: Amount,
```

Comment:
We should update the doc describing the `target_amount` parameter.

Reply:
Fixed.